image/image.go

```diff
@@ -26,6 +26,9 @@ import (
 // validates the manifest pointed to by the given ref
 // or returns an error if the validation failed.
 func ValidateLayout(src, ref string) error {
+	if _, err := os.Stat(src); err != nil {
```

Comment:
The code that is executed after this MUST be able to deal with the same situation (it's possible, even if unlikely, that that path is removed after this check but before being opened), which makes the check redundant. What's the benefit in returning early here?